util/regexp-assemble/data/932100.data

```diff
@@ -20,6 +20,8 @@
 
 ##! starting tokens prefix
 ##!> assemble
+##! time ifconfig
+time
```

Comment:
I think we might want to add the unix cmdline preprocessor here, otherwise it will be easily bypassed with classic techniques:
```suggestion
  ##!> cmdline unix
time
  ##!<
```

And repeating the same on the others below. Don't know if adding more tests, the cmdline preprocessor does its work pretty well...

Reply:
I updated all data files with the preprocessor.